source/client-side-encryption/etc/test-templates/bypassedCommand.yml.template

```diff
@@ -26,17 +26,18 @@ tests:
           command:
             ping: 1
           command_name: ping
-  - description: "current op is not bypassed"
+  - description: "kill op is not bypassed"
```

Comment:
The intent of this test seems to test that bypassed commands are passed (it tests with `ping` above) and then this test asserts that a command that isn't encrypted or bypassed errors.  

Replacing `currentOp` with `killOp` (another command that is neither bypassed or encrypted) tests the same scenario.